schemas/telemetry/usage/browser-features-schema.json

```diff
@@ -6,12 +6,16 @@
   "description": "Schema of browser specific features usage",
   "oneOf": [
     {
-      "required": ["feature"],
+      "required": ["feature", "is_forced"],
```

Comment:
If this field is `required`, is it possible to have retro-compatibility issues ?

Reply:
Indeed, we could also avoid to send this parameter when the value is false